drs3/dao/db.py

```diff
@@ -18,15 +18,18 @@
 from datetime import datetime
 from typing import Any, Optional
 
-from ghga_service_chassis_lib.postgresql import SyncPostgresqlConnector
+from ghga_service_chassis_lib.postgresql import (
+    PostgresqlConfigBase,
+    SyncPostgresqlConnector,
+)
 from ghga_service_chassis_lib.utils import DaoGenericBase
 from sqlalchemy.future import select
 
 from .. import models
-from ..config import config
+from ..config import CONFIG
 from . import db_models
 
-psql_connector = SyncPostgresqlConnector(config)
+psql_connector = SyncPostgresqlConnector(CONFIG)
```

Comment:
I think this can be removed since the SyncPostgresqlConnector is now defined in the PostgresDatabase class based on the config.